fields/types/azurefile/AzureFileType.js

```diff
@@ -144,7 +144,7 @@ azurefile.prototype.addToSchema = function () {
 		reset: function () {
 			try {
 				azure.createBlobService().deleteBlob(this.get(paths.container), this.get(paths.filename), function () {});
-			} catch (e) {}//eslint-disable-line no-empty
+			} catch (e) {}// eslint-disable-line no-empty
```

Comment:
there should be a space _before_ the comment as well. Not sure which rule should catch that though, possibly a different one?